packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -407,10 +407,16 @@ abstract class ScrollView extends StatelessWidget {
         return buildViewport(context, offset, axisDirection, slivers);
       },
     );
-    final Widget scrollableResult = primary && scrollController != null
+
+    Widget scrollableResult = primary && scrollController != null
         ? PrimaryScrollController.none(child: scrollable)
         : scrollable;
 
+    scrollableResult = OverscrollClipBehavior(
```

Comment:
This may not be necessary to add to the widget tree for every app, especially since the stretch is an opt-in on one platform. Could we instead pass the ScrollView.clipBehavior to Scrollable?
Then Scrollable can include it in the ScrollableDetails it provides to the indicator. :)

Reply:
Thanks for the feedback! Updated the PR.